base/int.jl

```diff
@@ -131,13 +131,40 @@ fld(x::Unsigned, y::Signed) = div(x, y) - (signbit(y) & (rem(x, y) != 0))
 
 """
     mod(x, y)
+    rem(x, y, RoundDown)
 
-Modulus after flooring division, returning in the range ``[0,y)``, if `y` is positive, or
-``(y,0]`` if `y` is negative.
+The reduction of `x` modulo `y`, or equivalently, the remainder of `x` after floored
+division by `y`. Returns a value with the same sign as `y` whose magnitude is less than
+`abs(y)` (except possibly in the case of floating point values, see note below).
 
+This has the property that
```

Comment:
Do you mean that, given `u = fld(x, y)` and `v = mod(x, y)`, computing the mul-add `u*y + v` without intermediate rounding yields `x`? Or do you mean something else? (Also, perhaps expand "This" to "This function"?)